runtime/src/main/jni/v8_inspector/src/inspector/v8-page-agent-impl.cpp

```diff
@@ -0,0 +1,257 @@
+//
+// Created by pkanev on 1/31/2017.
+//
+
+#include <v8_inspector/src/inspector/utils/v8-search-utils-public.h>
+#include "v8-page-agent-impl.h"
+#include "v8_inspector/src/inspector/utils/v8-page-resources.h"
+#include "search-util.h"
+
+namespace v8_inspector {
+
+namespace PageAgentState {
+static const char pageEnabled[] = "pageEnabled";
+}
+
+V8PageAgentImpl::V8PageAgentImpl(
+    V8InspectorSessionImpl* session, protocol::FrontendChannel* frontendChannel,
+    protocol::DictionaryValue* state)
+    : m_session(session), m_frontend(frontendChannel),
+      m_state(state),
+      m_enabled(false),
+      m_frameIdentifier("NSFrameIdentifier"),
```

Comment:
It's an arbitrary identifier that stands for NativeScriptFrameIdentifier. May as well call it Main, MainFrame, whatever, it is not visible anywhere but needs to be set nonetheless in the API.